keyboards/ergodox/readme.md

```diff
@@ -1,33 +1,38 @@
 3. Download the firmware file
 4. Connect the keyboard, press its Reset button (gently insert a paperclip into the hole in the top-right corner) and flash it using the Teensy loader you installed on step 1 and the firmware you downloaded.
 
-## More technical: create your own totally custom firmware by editing the source files. 
+## More technical: compile an existing keymap, or create your own totally custom firmware by editing the source files.
 
-This requires a little bit of familiarity with coding.
+This requires a little bit of familiarity with coding. 
+If you are just compiling an existing keymap and don't want to create your own, you can skip step 4, 5 and 8.
 
 1. Go to https://github.com/jackhumbert/qmk_firmware and read the readme at the base of this repository, top to bottom. Then come back here :)
 2. Clone the repository (download it)
 3. Set up a build environment as per [the build guide](/doc/BUILD_GUIDE.md) 
     - Using a Mac and have homebrew? just run `brew tap osx-cross/avr && brew install avr-libc`
-4. Copy `keyboards/ergodox_ez/keymaps/default/keymap.c` into `keymaps/your_name/keymap.c` (for example, `keymaps/german/keymap.c`)
+4. Copy `keyboards/ergodox/keymaps/default/keymap.c` into `keymaps/your_name/keymap.c` (for example, `keymaps/german/keymap.c`)
 5. Edit this file, changing keycodes to your liking (see "Finding the keycodes you need" below). Try to edit the comments as well, so the "text graphics" represent your layout correctly. See below for more tips on sharing your work.
-6. Compile your firmware by running `make keymap=your_name`. For example, `make keymap=german`. This will result in a hex file, which will be called `ergodox_ez_your_name.hex`, e.g. `ergodox_ez_german.hex`.
-6. Flash this hex file using the [Teensy loader](https://www.pjrc.com/teensy/loader.html) as described in step 4 in the "Easy Way" above. If you prefer you can automatically flash the hex file after successfull build by running `make teensy keymap=your_name`.
-7. Submit your work as a pull request to this repository, so others can also use it. :) See below on specifics.
+6. Compile your firmware by running `make keymap=keymap_name`. For example, `make keymap=german`. This will result in a hex file, which will be called `ergodox_ez_keymap_name.hex`, e.g. `ergodox_ez_german.hex`. For **Infinity ErgoDox** you need to add `subproject=infinity` to the make command.
+7. **ErgoDox EZ** - Flash this hex file using the [Teensy loader](https://www.pjrc.com/teensy/loader.html) as described in step 4 in the "Easy Way" above. If you prefer you can automatically flash the hex file after successful build by running `make teensy keymap=keymap_name`.
+
+   **Infinity ErgoDox** - Flash the firmware by running `make dfu-util keymap=keymap_name subproject=infinity`
```

Comment:
Currently it's not possible to do so from the make command directly. However you could run this command `make bin keymap=keymap_name subproject=infinity` which will generate a bin file ".build/ergodox_infinity_keymap_name.bin."

You can then flash this file manually using `dfu-util --serial <serial string> .build/ergodox_infinity_keymap_name.bin.

I can add support for this to the make file at some point.